cfgov/cfgov/settings/base.py

```diff
@@ -26,6 +26,11 @@
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 USE_X_FORWARDED_HOST = True
 
+# in some environments, we want to respect X-Forwarded-Port
+
+if 'USE_X_FORWARDED_PORT' in os.environ:
```

Comment:
See my latest comment on the Ansible PR; the way it's set up now, this could potentially be set to something like `false` and it'd still enable this here. The code as written now is only correct if we take care to _only_ define this variable when we want to forward the port.